documents/TestSuite/_options.mtlx

```diff
@@ -8,13 +8,19 @@
         <!-- List of comma separated file names acts as a filter to only test documents with these names -->
         <parameter name="overrideFiles" type="string" value="" />
 
+        <!-- List of comma separated file names which require color management  -->
+        <parameter name="cmsFiles" type="string" value="" />
```

Comment:
This is not the greatest but allows for some CMS load on demand in tests. Since it's empty for now the impl file is manually *not* loaded and the CMS system is manually *not* added to the generator.